app/_sections/Architecture.tsx

```diff
@@ -33,35 +33,36 @@ export default function Architecture() {
-                In essence, guards are dedicated to security maintenance, and executing responses, while watchers focus on the ongoing monitoring and transparent reporting."
+                description="Rosen is an Ergo-centric bridge fortified with multi-layered security protection. In the initial layer, Watchers monitor network activities and report valid bridge-related events to the subsequent layer, Guards. These Guards then carefully process the reported events and execute required actions. 
+                In brief, Guards are dedicated to security maintenance and executing responses, while Watchers are focused on the ongoing monitoring of activities and transparent reporting."
                 className="mt-16"
             />
             <StickyBackground/>
             <div className="block lg:w-3/4">
                 <img alt="Picture of Rosen architecture" src="./assets/architecture/rosen-architecture.png"/>
             </div>
-            <DescriptionBox title="Guard set" img="rosen-architecture-guardset.png">
-                <Text>Guards are a federated group of entities managing the Rosen core. Their authority over Rosen is restricted through multisignature contracts and wallets. Failure or collusion of guards will be tolerated while the majority of guards are healthy. Each guard has a reasonable amount of funds locked as collateral and will lose all of them at once in case of malicious behaviour.</Text>
+            <DescriptionBox title="Guard Set" img="rosen-architecture-guardset.png">
+                <Text>Guards are a federated group of entities managing the Rosen core. Their authority over Rosen is restricted through multisignature contracts and wallets. Failure or collusion of Guards will be tolerated while the majority of Guards are healthy. Each Guard has a reasonable amount of funds locked as collateral and will lose all their funds at once in case of malicious behaviour.</Text>
             </DescriptionBox>
             <DescriptionBox title="Watcher" img="rosen-architecture-watchers.png" right>
-                <Text>A watcher is an entity tasked with monitoring a blockchain and reporting bridge-related events on Ergo. Each blockchain has its own dedicated set of watchers working together as a cross-chain oracle for Rosen. Watchers will be rewarded for settled transfers but may face penalties for fraud. It’s open for anyone to join.</Text>
+                <Text>Watchers are entities tasked with monitoring a given blockchain (e.g., Cardano, Bitcoin, Ethereum, etc.) and reporting bridge-related events on Ergo. Each blockchain has its own dedicated set of Watchers working together as a cross-chain oracle for Rosen. Watchers will be rewarded for settled transfers but may face penalties for fraud. Anyone can participate as a Watcher once
+                     allocating enough RSN and native currency of the bridged chain (e.g., 1000 ERG and 1000 RSN).</Text>
```

Comment:
This sentence is incorrect. All watchers should put collateral in the form of ERG and RSN. Besides the collateral amount is not fixed yet.  
The collateral is explained in later sections and is not required here. This text part became a little long.

Reply:
I am not going to pretend to understand the intricacies of Rosen - I took what was there and tried to make it make more sense to the casual read (much like myself). If I was unable to do so, I think that it's pretty indicative there needs to be more context.

Does this make more sense? 

Watchers are entities tasked with monitoring a given blockchain (e.g., Cardano, Bitcoin, Ethereum, etc.) and reporting bridge-related events on Ergo. Each blockchain has its own dedicated set of Watchers working together as a cross-chain oracle for Rosen. Watchers will be rewarded for settled transfers but may face penalties for fraud. Anyone can participate as a Watcher once allocating enough RSN and and ERG).</Text>

Reply:
Yes its much better